NBitcoin.Tests/util_tests.cs

```diff
@@ -183,7 +183,7 @@ public void CanParseRootedKeyPath()
 			Assert.Equal("7b09d780", result.MasterFingerprint.ToString());
 			Assert.True(RootedKeyPath.TryParse("7b09d780/", out result));
 			Assert.Equal("7b09d780", result.MasterFingerprint.ToString());
-			Assert.Equal("7b09d780/", result.ToString());
+			Assert.Equal("7b09d780", result.ToString());

```

Comment:
we should not change that.

Reply:
Fixed in https://github.com/MetacoSA/NBitcoin/pull/696/commits/2d521d4817a3b0621988c1113311aada38d0546f